<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1478</width>
    <height>968</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::TabPosition::North</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="elideMode">
       <enum>Qt::TextElideMode::ElideLeft</enum>
      </property>
      <widget class="QWidget" name="StartStop">
       <attribute name="title">
        <string>Tab 1 (Start/Stop)</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="Bild.qrc">:/Herunterladen.png</pixmap>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="3" column="0">
           <widget class="QPushButton" name="startButton">
            <property name="text">
             <string>Start</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QPushButton" name="Starten">
            <property name="text">
             <string>IviumSoft-Starten</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="4">
           <widget class="QTableWidget" name="tableWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="sizeAdjustPolicy">
             <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustIgnored</enum>
            </property>
            <attribute name="horizontalHeaderVisible">
             <bool>true</bool>
            </attribute>
            <attribute name="horizontalHeaderCascadingSectionResizes">
             <bool>true</bool>
            </attribute>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>250</number>
            </attribute>
            <attribute name="horizontalHeaderHighlightSections">
             <bool>true</bool>
            </attribute>
            <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
             <bool>true</bool>
            </attribute>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <column>
             <property name="text">
              <string>Nr.</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Zeit [s]</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Channel</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Strom [mA] </string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Spannung [V]</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Q_N [mAh]</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Q_0 [mAh]</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Q(t)</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Q_min</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Anzahl_Zyklen</string>
             </property>
            </column>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QPushButton" name="delButton">
            <property name="text">
             <string>Delete</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2" colspan="2">
           <widget class="QPushButton" name="Beenden">
            <property name="text">
             <string>IviumSoft-Beenden</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QPushButton" name="copyButton">
            <property name="text">
             <string>Copy</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="stopButton">
            <property name="text">
             <string>Stop</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Channel-Auswahl</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDial" name="dial">
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="maximum">
               <number>20</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox">
              <property name="text">
               <string>Aktivieren</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcdNumber">
            <property name="frameShape">
             <enum>QFrame::Shape::Box</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Raised</enum>
            </property>
            <property name="smallDecimalPoint">
             <bool>false</bool>
            </property>
            <property name="digitCount">
             <number>5</number>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::SegmentStyle::Filled</enum>
            </property>
            <property name="value" stdset="0">
             <double>1.000000000000000</double>
            </property>
            <property name="intValue" stdset="0">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcdNumber_2">
            <property name="digitCount">
             <number>5</number>
            </property>
            <property name="value" stdset="0">
             <double>1.000000000000000</double>
            </property>
            <property name="intValue" stdset="0">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Intervall in [Sec]</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDial" name="dial_Zeitintervall">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>60</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <widget class="QRadioButton" name="radioButton_Zeitformat_1">
                <property name="text">
                 <string>Zeit in &quot;tt&quot;</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
                <property name="autoRepeat">
                 <bool>true</bool>
                </property>
                <property name="autoRepeatInterval">
                 <number>1</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioButton_Zeitformat_2">
                <property name="text">
                 <string>Zeit in &quot;hh:mm:tt:zzz&quot;</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="SOCSOH">
       <attribute name="title">
        <string>Tab 2  (SOC/SOH)</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="5" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QSpinBox" name="ProzentAnzeigeSOH">
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::ButtonSymbols::UpDownArrows</enum>
            </property>
            <property name="suffix">
             <string>%</string>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="BatterieAnzeigeSOH">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="value">
             <number>24</number>
            </property>
            <property name="textVisible">
             <bool>false</bool>
            </property>
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="invertedAppearance">
             <bool>false</bool>
            </property>
            <property name="textDirection">
             <enum>QProgressBar::Direction::TopToBottom</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="SohButton">
            <property name="text">
             <string>SOH-berechnen</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QTextEdit" name="textAusgabe"/>
        </item>
        <item row="0" column="0" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="Bild.qrc">:/Herunterladen.png</pixmap>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="5" column="1">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QSpinBox" name="ProzentAnzeigeSOC">
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::ButtonSymbols::UpDownArrows</enum>
            </property>
            <property name="suffix">
             <string>%</string>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="BatterieAnzeige">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="value">
             <number>24</number>
            </property>
            <property name="textVisible">
             <bool>false</bool>
            </property>
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="invertedAppearance">
             <bool>false</bool>
            </property>
            <property name="textDirection">
             <enum>QProgressBar::Direction::TopToBottom</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="SocButton">
            <property name="text">
             <string>SOC-berechnen</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0" colspan="2">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_Q_0">
            <property name="text">
             <string> N_Zyklen Formierung; eingeben und mit Enter bestätigen</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="pushButtonNZyklen">
              <property name="text">
               <string>Enter</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="doubleSpinBox_NZyklen">
              <property name="suffix">
               <string/>
              </property>
              <property name="decimals">
               <number>2</number>
              </property>
              <property name="singleStep">
               <double>0.010000000000000</double>
              </property>
              <property name="value">
               <double>0.000000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="Anzeige_NZyklen">
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::ButtonSymbols::UpDownArrows</enum>
              </property>
              <property name="suffix">
               <string/>
              </property>
              <property name="decimals">
               <number>2</number>
              </property>
              <property name="value">
               <double>0.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="1" column="0" colspan="2">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_Q_N">
            <property name="text">
             <string> Q_N eingeben und mit Enter bestätigen</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="buttonNennSpg">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Enter</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="doubleSpinBoxNennSpg">
              <property name="suffix">
               <string> Ah</string>
              </property>
              <property name="decimals">
               <number>2</number>
              </property>
              <property name="singleStep">
               <double>0.010000000000000</double>
              </property>
              <property name="value">
               <double>0.000000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="AnzeigeNspg">
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::ButtonSymbols::UpDownArrows</enum>
              </property>
              <property name="suffix">
               <string> Ah</string>
              </property>
              <property name="decimals">
               <number>2</number>
              </property>
              <property name="value">
               <double>0.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QRadioButton" name="QNManuellButton">
            <property name="text">
             <string>&quot;Enter&quot; aktivieren/deaktivieren</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1478</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="action_ffnen"/>
    <addaction name="actionSchlie_en_2"/>
    <addaction name="actionspeichern"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <action name="actionSchlie_en">
   <property name="text">
    <string>Schließen</string>
   </property>
  </action>
  <action name="action_ffnen">
   <property name="text">
    <string>Öffnen</string>
   </property>
  </action>
  <action name="actionSchlie_en_2">
   <property name="text">
    <string>Schließen</string>
   </property>
  </action>
  <action name="actionspeichern">
   <property name="text">
    <string>Speichern</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Bild.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>dial_Zeitintervall</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_2</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>879</x>
     <y>617</y>
    </hint>
    <hint type="destinationlabel">
     <x>79</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
